Mismatched types

On Alpine Linux (using musl libc instead of GNU libc):

```
error[E0308]: mismatched types=======> ] 411/412: shai(bin)                                                                                                                                                                                  
    --> shai-cli/src/shell/pty.rs:221:43
     |
221  |             if libc::ioctl(self.slave_fd, libc::TIOCSCTTY as libc::c_ulong, 0) == -1 {
     |                -----------                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `i32`, found `u64`
     |                |
     |                arguments to this function are incorrect
     |
note: function defined here
    --> /home/alpine/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/libc-0.2.174/src/unix/linux_like/mod.rs:1779:20
     |
1779 |             pub fn ioctl(fd: c_int, request: Ioctl, ...) -> c_int;
     |                    ^^^^^
help: you can convert a `u64` to an `i32` and panic if the converted value doesn't fit
     |
221  |             if libc::ioctl(self.slave_fd, (libc::TIOCSCTTY as libc::c_ulong).try_into().unwrap(), 0) == -1 {
     |                                           +                                +++++++++++++++++++++

For more information about this error, try `rustc --explain E0308`.
```
I suspect this is something related to the libc flavour.

--- a/shai-cli/src/shell/pty.rs
+++ b/shai-cli/src/shell/pty.rs
@@ -218,7 +218,7 @@ impl ShaiPtyManager {
             
             libc::setsid();
             
-            if libc::ioctl(self.slave_fd, libc::TIOCSCTTY as libc::c_ulong, 0) == -1 {
+            if libc::ioctl(self.slave_fd, libc::TIOCSCTTY.try_into().unwrap(), 0) == -1 {
                 libc::exit(1);
             }
             

